pipeline/dbt/models/staging/sources/fredo/stg_fredo__structures.sql

```diff
@@ -4,12 +4,14 @@ WITH source AS (
 
 final AS (
     SELECT
+        _di_source_id                                         AS "_di_source_id",
         data ->> 'id'                                         AS "id",
         NULLIF(TRIM(data ->> 'adresse'), '')                  AS "adresse",
         NULLIF(TRIM(data ->> 'code_postal'), '')              AS "code_postal",
         NULLIF(TRIM(data ->> 'commune'), '')                  AS "commune",
         NULLIF(TRIM(data ->> 'frais'), '')                    AS "frais",
         NULLIF(TRIM(data ->> 'horaires_ouverture'), '')       AS "horaires_ouverture",
+        NULLIF(TRIM(data ->> 'last_update'), '')              AS "last_update",
```

Comment:
tu peux nettoyer le champ en staging

```suggestion
        CAST(NULLIF(TRIM(data ->> 'last_update'), '') AS DATE)              AS "last_update",
```

voir même utiliser TO_DATE